subset/network/ntp_tests.py

```diff
@@ -63,9 +63,13 @@ def ntp_payload(packet):
 
 
 def test_ntp_support():
-    capture = rdpcap(startup_pcap_file)
-    if len(capture) > 0:
-        version = ntp_client_version(capture)
+    startup_capture = rdpcap(startup_pcap_file)
```

Comment:
you shouldn't actually use startup_pcap_file at all, unless you are explicitly trying to track something during the DHCP exchange.  If there is no monitor_pcap then just return false.